apps/Notejot/install-64

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
```

Comment:
Please add `install_packages flatpak || exit 1`

Reply:
I added install_packages to `install-64`, as well as `purge_packages` to the uninstall script I assume that's what you want..